loops.js

```diff
@@ -50,4 +64,14 @@
 // >> 8
 // >> 9
 // ==========================================
-
+for (i = 0; i <= 9; i++) {
```

Comment:
<=9  werkt goed, maar ik volgens mij is het gebruikelijker en korter om < 10 te gebruiken